mesheryctl/internal/cli/root/system/model.go

```diff
@@ -128,10 +129,40 @@ mesheryctl system model list --page 2
 		}
 
 		if len(rows) == 0 {
-			// if no model is found
 			utils.Log.Info("No model(s) found")
-		} else {
+			return nil
+		}
+
+		if cmd.Flags().Changed("page") {
 			utils.PrintToTable(header, rows)
+		} else {
+			paginator := utils.NewPaginator(header, rows)
+			paginator.Render()
+			keysEvents, err := keyboard.GetKeys(10)
+			if err != nil {
+				panic(err)
```

Comment:
Please use MeshKit-style error handling, @zakisk

Reply:
No need to panic @zakisk 😉

Reply:
@zakisk still working on this?